ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -108,8 +108,13 @@ fi
 param set MAV_SYS_ID $((px4_instance+1))
 simulator_tcp_port=$((4560+px4_instance))
 udp_offboard_port_local=$((14580+px4_instance))
-udp_offboard_port_remote=$((14540+px4_instance))
-udp_gcs_port_local=$((14570+px4_instance))
+if [ $px4_instance > 9]
+	udp_offboard_port_remote=14549
+then
+	udp_offboard_port_remote=$((14540+px4_instance))
+fi
```

Comment:
```
udp_offboard_port_remote=$((14540+px4_instance))
[ $px4_instance > 9] && udp_offboard_port_remote=14549 # use the same ports for more than 10 instances to avoid port overlaps
```

Reply:
Thanks! that is much cleaner